docs/antora/modules/ROOT/pages/Common_Project_Layouts.adoc

```diff
@@ -92,8 +92,15 @@ import mill._
 import mill.scalajslib._
 
 object foo extends ScalaJSModule {
-  def scalaVersion = "2.12.4"
-  def scalaJSVersion = "0.6.22"
+  def scalaVersion = "2.13.8"
+  def scalaJSVersion = "1.10.1"
+  // Make the DOM available by using the jsdom library (as in Scala.js tutorial)
+  def jsEnvConfig = T(scalajslib.api.JsEnvConfig.JsDom())
+  def ivyDeps = Agg(ivy"org.scala-js::scalajs-dom::2.2.0")
```

Comment:
I would first `import mill.scalajslib.api._` at the beginning of the file and then write `def jsEnvConfig = T(JsEnvConfig.JsDom())`
What do you think?

At the same time, `jsdom` needs to be installed with npm and has different complications, so I don't know if it should be referenced in the first Scala.js example.

Reply:
Humm, I agree on npm dependency but I thought about adding it since it's not documented anywhere making a newcomer life harder if starting with the Scala.js tutorial and using Mill...

What do you think? Remove it or do something like linking to the tutorial?